python/ray/tests/autoscaler/providers.py

```diff
@@ -0,0 +1,24 @@
+from ray.autoscaler._private.providers import (
```

Comment:
Do we need to add it to BUILD file so it actually runs as test?

Reply:
added, let's see

Reply:
I think that made the tests pass!